client/rest-high-level/src/test/java/org/elasticsearch/client/RequestTests.java

```diff
@@ -1247,6 +1247,11 @@ public void testRankEval() throws Exception {
                 new PrecisionAtK());
         String[] indices = randomIndicesNames(0, 5);
         RankEvalRequest rankEvalRequest = new RankEvalRequest(spec, indices);
+        String wildCardOption = randomFrom("none", "open", "closed", "all");
+        String ignoreUnavailableOption = randomFrom("true", "false");
+        String allowNoIndicesOption = randomFrom("true", "false");
+        rankEvalRequest.indicesOptions(IndicesOptions.fromParameters(wildCardOption,
+                ignoreUnavailableOption, allowNoIndicesOption, SearchRequest.DEFAULT_INDICES_OPTIONS));
```

Comment:
we have a method for this I think, `setRandomIndicesOptions`

Reply:
+1